### PR TITLE
fix #108

### LINKED ISSUE #108: SendHTML Question
Hi Guys, 

I have a short question concerning, concerning how to use Polaris to send already generated HTML. Abviously, I am doing something wrong, but I can't figure out what.

This is my example code:

```powershell

New-PolarisRoute -Path /sendhtml -Method GET -ScriptBlock {
    $file= "C:\MyHtml\HTML_Example.html"  
    $htmlContent = gc $file
    $reponse.sendhtml($htmlContent)

}

$app = Start-Polaris -Port 8082 -MinRunspaces 1 -MaxRunspaces 5 -UseJsonBodyParserMiddleware -Verbose # all params are optional

#Didnt work

```

When I open my browser, and navigate to http://localhost:8082/sendhtml It take quite some while, but after some 20 seconds, it sends me a file called 'sendhtml', which is not the content that I had in my html_Example.

I tried the following different ways to try to send html back, all without success:

```powershell

New-PolarisRoute -Path /send -Method GET -ScriptBlock {
    $file= "C:\MyHtml\HTML_Example.html"  
    $htmlContent = gc $file
    $reponse.send$htmlContent)

}

$app = Start-Polaris -Port 8082 -MinRunspaces 1 -MaxRunspaces 5 -UseJsonBodyParserMiddleware -Verbose # all params are optional

#Didnt work as well

```

```powershell

$myhtmlScript = "C:\Scripts\example01.ps1" #this script generates HTML on the fly
$htmlcontent = gc $myhtmlScript
$content = [scriptblock]::Create($htmlcontent) #gc $myhtmlScript

New-polarisRoute -Path /sb -method GET -ScriptBlock $content #did not work
New-PolarisRoute -Path /pshtml -Method GET -ScriptPath $myhtmlScript #did not work

$app = Start-Polaris -Port 8082 -MinRunspaces 1 -MaxRunspaces 5 -UseJsonBodyParserMiddleware -Verbose # all params are optional

```

## error behaviour

The error behaviour is 'the same' for all the routes above: It takes some time to load (20 sec or so) and sends me a file with the name of the route back, which seems to be empty ($null).

Can you tell me what I am doing wrong, or is this perhaps not supported yet?

Thanks

### PR DESCRIPTION
**Pull request by Stephanevg**
Fix for issue #108 where .send() would not send content back if the route method was not set to an upper case http method.
This fix allows the user to have the flexibility to use either lower, or upper case when it creates the routes (under the hoods, it will simply do a .ToUpper() )

### CODE REVIEW

File: Tests/New-PolarisRoute.Tests.ps1 [hunk 1]
```diff
@@ -23,6 +23,22 @@
...
+            New-PolarisRoute -Path /108 -Method get -ScriptBlock {
+                
+                $line = "<h1>this is H1</h1>"
+                $response.SetContentType("text/html");  
+                $response.send($line)
+            } -force
+            
+            (Get-PolarisRoute -Path 108).Method | should be 'GET'
+        }
+
```
Comment (tylerl0706) [inline Tests/New-PolarisRoute.Tests.ps1]: nit pick, the indentation seems a bit off and there looks like an extra line at the top and bottom (should have 1 line empty between each test).

Can you just do a quick touch up of that? 😃

Comment (Stephanevg) [reply Tests/New-PolarisRoute.Tests.ps1]: sure!

I made the changes as requested. Let me know if something else is needed

**Comment by Stephanevg** [comment]
Ok, I did the formating changes, and incremented the module version to 0.1.1
Hope all is ok now. Let me know!

**Comment by tylerl0706** [comment]
Merge time 👍 thank you once again for contributing!

**Pull request merged**